### PR TITLE
Make participant model searchable

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: operations/views.py [hunk 1]
```diff
@@ -44,6 +44,20 @@ class ParticipantViewSet(viewsets.ModelViewSet):
     serializer_class = ParticipantSerializer
 
 
+class ParticipantSearchList(generics.ListAPIView):
```
Comment (imsickofmaps) [inline operations/views.py]: You don't need any of this really. You can just add `filter_fields = ('msisdn',)` to `ParticipantViewSet` and then `api/v1/participants/?msisdn=%2B27820010001` will work

**Comment by gsvr** [comment]
@imsickofmaps You're right. Happy with this?

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**